docs/usage/self-hosted-configuration.md

```diff
@@ -470,6 +470,11 @@ Use the `globalExtends` field if your preset has any global-only configuration o
 
 Use the `extends` field instead of this if, for example, you need the ability for a repository config (e.g. `renovate.json`) to be able to use `ignorePresets` for any preset defined in global config.
 
+<!-- prettier-ignore -->
+!!! warning
+    `globalExtends` presets can't be private.
+    TODO FOR MAINTAINERS: please explain _why_ it can't be private, or what resources must be public.
```

Comment:
Can one of the maintainers please expand on this? To me it's not really clear what we mean. What resources must be public for `globalExtends` to work?

Do we mean to say: "Only point `globalExtends` to presets that are hosted on public repositories"?

Reply:
at the time of resolving global extends, the config isn't fully processed, so no auth is available to fetch private things

Reply:
```suggestion
    `globalExtends` presets can't be private.
    When Renovate resolves `globalExtends` it does not fully process the configuration.
    This means that Renovate does not have the authentication it needs to fetch private things.
```

How about this then?